src/arbiter.rs

```diff
@@ -32,20 +32,48 @@ thread_local!(
 /// can belongs to specific `System` actor.
```

Comment:
Add few lines about panic behavior for system arbiter and user created ones

Reply:
Done.